plugins/bitrise/README.md

```diff
@@ -0,0 +1,66 @@
+# Bitrise
+
+Welcome to the Bitrise plugin!
+
+- View recent Bitrise Builds for a Bitrise application
+- Download build artifacts
+
+## Installation
+
+```sh
+$ yarn add @backstage/plugin-bitrise
+```
+
+Then make sure to export the plugin in your app's [`plugins.ts`](https://github.com/backstage/backstage/blob/master/packages/app/src/plugins.ts) to enable the plugin:
+
+```js
+export { bitrisePlugin } from '@backstage/plugin-bitrise';
+```
+
+Modify your app routes in `App.tsx` to include the entry component exported from the plugin, for example:
+
+```tsx
+// At the top imports
+import { EntityBitrisePage } from '@backstage/plugin-bitrise';
+
+// Inside App component
+<Routes>
+  // ...
+  <Route path="/bitrise" element={<EntityBitrisePage />} />
+  // ...
+</Routes>;
+```
```

Comment:
I'm not sure about this one. I would expect that the `EntityBitrisePage` should be added to the [`EntityPage.tsx`](https://github.com/backstage/backstage/blob/eba975b693a2c40cd0afffeb82c94ff6367f41e4/packages/app/src/components/catalog/EntityPage.tsx#L108) like this?
```ts
    <Grid item xs={12}>
      <EntityBitrisePage />
    </Grid>
```

or as part of the [`CICDSwitcher`](https://github.com/backstage/backstage/blob/eba975b693a2c40cd0afffeb82c94ff6367f41e4/packages/app/src/components/catalog/EntityPage.tsx#L108).

I would assume that the `EntityBitrisePage` will crash on the internal `useEntity` hook since it is not in the correct context what added to the core router.

Reply:
I adjusted the documentation and renamed the component to `<EntityBitriseContent />` 👍